### DISCUSSION TITLE
How to specify OneDrive folders

### DISCUSSION THREAD

**Discussion by speedster-pta** [abraunegg/onedrive#1503] [Q&A] [answered]
I might be missing the obvious, but after scouring the configuration documents I can't find how to select which online folders to sync? Setting up the local folders / files is well documented but I have a couple of online folders that shouldn't be synced to the local client but I can't quite figure out how to exclude them?

**Comment by abraunegg** [comment, marked as answer]
@speedster-pta 
> I might be missing the obvious, but after scouring the configuration documents I can't find how to select which online folders to sync?

The same configuration options are applicable regardless of sync direction. The options 'skip_dir', 'skip_file' and 'sync_list' are applied in the following sync operations:
* OneDrive -> local
* local -> OneDrive
